test/index.js

```diff
@@ -1,22 +1,49 @@
 const expect = require('expect.js'),
-  init = require('../index');
+  init = require('../index'),
+  sinon = require('sinon');
+
 
 describe('beautiful-logs', function() {
   it('should expose a function', function() {
     expect(init).to.be.a('function');
   });
+
   describe('logger', function() {
     describe('.addCallback()', function() {
       it('should be a function', () => expect(init().addCallback).to.be.a('function'));
-      it('added callbacks should be called on log events', function(done) {
+
+      const generateTest = (level) => (done) => {
         const logger = init();
         logger.addCallback(function(event) {
-          expect(event).to.have.property('level', 'info');
+          expect(event).to.have.property('level', level);
           expect(event).to.have.property('message', 'Test message!');
           done();
         });
-        logger.info('Test message!');
+        logger[level]('Test message!');
+      }
+
+      const levels = ['trace', 'debug', 'info', 'warn', 'err', 'fatal'];
+      levels.forEach(level => {
+        it(`should be called ${level}`, generateTest(level));
       });
     });
+
+    describe('.lineEmpty()', function() {
+      it('should be called once', function() {
+          const logger = init();
+          sinon.spy(logger, 'emptyLine');
+          logger.emptyLine('trace', 'Test message!');
+          expect(logger.emptyLine.calledOnce).to.be.true;
```

Comment:
It doesn't make any sense to spy on the .empyLine() function. `sinon.spy(console,'log')` (if that is the correct way) is the better solution.